src/reducers/chatReducer.js

```diff
@@ -11,12 +11,12 @@ const initialState = {
     statuses: []
 };
 
-export default chatReducer = (state = initialState, action) => {
+function chatReducer(state = initialState, action) {
     switch (action.type) {
 
         case CHAT_ADD_MESSAGE:
             return Object.assign({
-                messages: [...state.messages, {
+                messages: [...state.emessages, {
```

Comment:
That is fixed. Weird though, i found the mispelling while i was testing locally and fixed it before i sent the PR. Or so i thought. Must have done something wrong.